### PR TITLE
59 update READMEs

### LINKED ISSUE #59: Make the READMEs nice
Make sure at least the main README is nice and clear for anyone visiting this repo after we do the project update on 16th Sept

The main README could do with a bit of extra info about the project and a quick summary of what the repo folders have in them (and links to them).
I think the pipeline README is outdated now, but also I think it needs a really clear record of the order in which you run the run_script files in - and what exactly the inputs and output file names you used were (you could make it clear these are changeable, but it might be nice to have a record). "step 1... run x, step 2 .. run y". The pipeline README could also have a short summary of the evaluation results.

### PR DESCRIPTION
**Pull request by crispy-wonton**
Fixes #59 

# Description

Update READMEs to contain all relevant and required information for dissemination including:
- citations of all data used
- attribution statements of external data
- instructions on running pipeline
- description of repo purpose and structure
- description of heat pump suitability calculations

3 READMEs updated:
- main README - general project information
- config/README - data sources
- pipeline/README - instructions to run pipeline

A couple of small bugs fixed along the way.

# Instructions for Reviewer

Please could you:
- check READMEs contain all required information
- check all sources are included in config/README

# Checklist:

- [ ] I have refactored my code out from `notebooks/`
- [x] I have checked the code runs
- [ ] I have tested the code
- [x] I have run `pre-commit` and addressed any issues not automatically fixed
- [x] I have merged any new changes from `dev`
- [x] I have documented the code
  - [x] Major functions have docstrings
  - [x] Appropriate information has been added to `README`s
- [x] I have explained this PR above
- [x] I have requested a code review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,98 @@
...
+| Is there a high property density (>60 households per km2) in this LSOA?           | 0        | 0        | 0        | 0        | 2       | 2       | 0      | 0      |
+| Is this property in an urban LSOA/high heat demand density LSOA?                  | 0        | 0        | 0        | 0        | 0       | 0       | 2      | 2      |
+| Maximum points per property                                                       | 4        | 4        | 4        | 3        | 7       | 5       | 5      | 5      |
+
+## Data sources and acknowledgements
+
+A comprehensive table of citations for data used in this analysis can be found in [asf_heat_pump_suitability/config/README.md](https://github.com/nestauk/asf_heat_pump_suitability/tree/dev/asf_heat_pump_suitability/config#readme).
+
+Contains OS data © Crown copyright and database right 2023.\
+Contains Royal Mail data © Royal Mail copyright and database right 2023.\
```
Comment (danlewis85) [inline README.md]: Year should be 2024.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,98 @@
...
+Contains Office for National Statistics information licensed under the Open Government Licence v.3.0.\
+Contains public sector information licensed under the Open Government Licence v3.0.
+
```
Comment (danlewis85) [inline README.md:93]: Add the following information for the land registry:

This work uses HM Land Registry's INSPIRE Index Polygons service. This information is subject to Crown copyright and database rights 2024 and is reproduced with the permission of HM Land Registry.

The polygons (including the associated geometry, namely x, y co-ordinates) are subject to Crown copyright and database rights 2024 Ordnance Survey 100026316.

INSPIRE Index Polygons service [Conditions of use](https://use-land-property-data.service.gov.uk/datasets/inspire#conditions)

File: README.md [hunk 3]
```diff
@@ -1,12 +1,98 @@
...
+This work uses [Facebook Research's balance package](https://github.com/facebookresearch/balance) and [ipfn](https://github.com/Dirguis/ipfn) to conduct iterative proportional fitting.
```
Comment (danlewis85) [inline README.md:94]: Add the following information for the Microsoft Building Footprints data:

Microsoft [GlobalMLBuildingFootprints](https://github.com/microsoft/GlobalMLBuildingFootprints) are made available under the Open Database License: [http://opendatacommons.org/licenses/odbl/1.0/](https://opendatacommons.org/licenses/odbl/1.0/). Any rights in individual contents of the database are licensed under the Database Contents License: [http://opendatacommons.org/licenses/dbcl/1.0/](https://opendatacommons.org/licenses/dbcl/1.0/)

File: README.md [hunk 4]
```diff
@@ -1,12 +1,98 @@
...
+| Maximum points per property                                                       | 4        | 4        | 4        | 3        | 7       | 5       | 5      | 5      |
+
+## Data sources and acknowledgements
+
+A comprehensive table of citations for data used in this analysis can be found in [asf_heat_pump_suitability/config/README.md](https://github.com/nestauk/asf_heat_pump_suitability/tree/dev/asf_heat_pump_suitability/config#readme).
+
+Contains OS data © Crown copyright and database right 2023.\
+Contains Royal Mail data © Royal Mail copyright and database right 2023.\
+Contains Office for National Statistics information licensed under the Open Government Licence v.3.0.\
+Contains public sector information licensed under the Open Government Licence v3.0.
```
Comment (danlewis85) [inline README.md]: Add:

Contains GeoPlace data © Local Government Information House Limited copyright and database right 2024

Which I _think_ is required to cover OS address products (e.g. uprn lookups).

File: README.md [hunk 5]
```diff
@@ -1,12 +1,98 @@
-# heat_pump_suitability
+# asf_heat_pump_suitability
+
+Calculate heat pump suitability scores for lower-layer super output areas (LSOAs) in England and Wales using domestic
+EPC data and supplementary sources. Scores are weight-adjusted for LSOAs where possible to reduce bias.
+Intermediate outputs include:
```
Comment (lizgzil) [inline README.md]: A gentler intro might be nicer here, e.g.
```suggestion
# Heat Pump Suitability

The `asf_heat_pump_suitability` repo contains the code used to calculate heat pump suitability scores for lower-layer super output areas (LSOAs) in England and Wales using domestic EPC data and supplementary sources. Scores are weight-adjusted for LSOAs where possible to reduce bias. Read more about the project [here](https://www.nesta.org.uk/project/mapping-heat-pump-suitability-across-great-britain/) . 

Intermediate outputs include:
```

File: README.md [hunk 6]
```diff
@@ -1,12 +1,98 @@
...
+
+- EPC data weighted according to LSOA using Iterative Proportional Fitting to reduce bias
+- EPC data enhanced with new features including: lat/lon; listed building status; building conservation zone status;
+  off gas status; average garden size per MSOA; property density per LSOA
+- Individual garden size estimates for UPRNs in EPC
 
 ## Setup
```
Comment (lizgzil) [inline README.md:8]: Think it'd be good to put this section at the end, just before the contributor guidelines section. It's a bit off putting for anyone coming to the repo to only look at the code.

File: asf_heat_pump_suitability/pipeline/README.md [hunk 7]
```diff
@@ -1,25 +1,120 @@
...
+
+   **Purpose**: Add LSOA & MSOA data to each EPC row and weight each EPC property according to LSOA with Iterative Proportional Fitting.
+
+   **Run**:
+
+   `python asf_heat_pump_suitability/pipeline/run_scripts/run_compute_epc_weights.py --epc_path s3://asf-daps/lakehouse/processed/epc/deduplicated/processed_dedupl-0.parquet -y 2023 -q 4`
+
+   **Inputs**: preprocessed deduplicated EPC dataset
+
+   **Outputs**:
```
Comment (lizgzil) [inline asf_heat_pump_suitability/pipeline/README.md:37]: this format is so lovely and clear! ✨

**Review by lizgzil — COMMENTED** [review]
thanks for this Roisin! It looks good to me and great to have all this extra info, I just had some suggestions for the landing README.

File: README.md (hunk 1 above)
Comment (crispy-wonton) [reply README.md]: We used 2023 data. Should it still be 2024?

File: README.md (hunk 4 above)
Comment (crispy-wonton) [reply README.md]: cool, added, I also have these in the config/README.md for OS data:

> 1. Contains OS data © Crown copyright and database right 2023.
> 2. Contains Royal Mail data © Royal Mail copyright and database right 2023.

File: README.md (hunk 1 above)
Comment (danlewis85) [reply README.md]: Yes. It's a bit of a grey area and some people would use the date that the dataset was originally created. However, given the possibility of changes etc. I'd prefer to use the date that the data was accessed, which is this year.

**Pull request merged**